gateway/api/server.go

```diff
@@ -447,4 +448,20 @@ func (api *Api) buildRoutes(route *gin.RouterGroup) {
 		api.AllowApiKey,
 		api.Authenticate,
 		apiserverinfo.New(api.GrpcURL).Get)
+
+	// Jira Integration routes
+	route.GET("/jira-integration",
+		AdminOnlyAccessRole,
+		api.Authenticate,
+		apijiraintegration.Get)
+
+	route.POST("/jira-integration",
+		AdminOnlyAccessRole,
+		api.Authenticate,
+		apijiraintegration.Post)
+
+	route.PUT("/jira-integration",
```

Comment:
I'd change these routes to `/integrations/jira`. It allows for extensibility with integrations and makes it more semantic with rest api